addon/components/sl-menu.js

```diff
@@ -555,7 +501,7 @@ export default Ember.Component.extend({
         }, this );
 
         return currentIndex;
-    }.property().volatile(),
+    }).volatile(),
```

Comment:
This is still respected with the inclusion and use of ember-disable-prototype-extensions?